### ISSUE TITLE
Treatment for hidden media

### ISSUE THREAD

**Issue by pleary** [inaturalist/INaturalistIOS#670]
The website recently released a feature where curators [can hide photos and sounds](https://github.com/inaturalist/inaturalist/issues/3803) by using ModeratorActions. On the website we replace the media with a message saying "Content Hidden" which users can click to open a popover showing who hid the content, when it was hidden, and the message left by the curator as to why it was hidden. Ideally mobile apps implement at least the "Content Hidden" message for hidden media.

I made a test observation https://www.inaturalist.org/observations/185148530 which has a hidden photo, a hidden sound, as well as a hidden identification, and a hidden comment (these latter two will have separate tickets). In the [API response for this observation](https://api.inaturalist.org/v1/observations/185148530), the media have an attribute `hidden: true`. They also have an attribute `moderator_actions` which contains information on the hiding action including the hiding user, date it was hidden, and the `reason` message added by the curator when hiding the content. Similar to images hidden for copyright violations, image URLs for hidden images are [replaced with a placeholder](https://www.inaturalist.org/assets/media-hidden-square.png) saying the media was hidden. This is a fixed replacement image with English text that is not internationalized.

Ideally hidden media would be treated as a special case, rendering a message similar to "Content Hidden", that can be localized for the viewing user. If it makes sense to do so and there is room, the hiding user, date of hiding, and hiding `reason` could also be displayed somewhere. The hiding reason could be long, and on the web we only show it in a popover.

In summary:
- [ ] for observation media that have an attribute `hidden: true`, do not render the image represented by their `url`, rather display a message `Content Hidden` that can be localized to the viewing user's locale
- [ ] (optional) optionally include somewhere information about the user that hid the media (i.e. created the `moderator_action`), the date the media was hidden, and the reason the media was hidden
